server/backend/ios_notifications/ios_crypto/encryptedString.go

```diff
@@ -0,0 +1,76 @@
+// Package ios_crypto provides functions for encrypting and decrypting strings using AES-256-GCM.
+package ios_crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/base64"
+)
+
+type EncryptedString string
+
+func (e *EncryptedString) String() string {
+	return string(*e)
+}
+
+func SymmetricEncrypt(plaintext string, key string) (*EncryptedString, error) {
+	bytesKey := []byte(key)
+	bytesPlaintext := []byte(plaintext)
+
+	c, err := aes.NewCipher(bytesKey)
+
+	if err != nil {
```

Comment:
```suggestion
	c, err := aes.NewCipher(bytesKey)
	if err != nil {
```

Please condense your code in those cases.
vspace is always limited ;)